ImageViewer/Source/GalleryViewController.swift

```diff
@@ -149,16 +149,17 @@ open class GalleryViewController: UIPageViewController, ItemControllerDelegate {
 
         NotificationCenter.default.removeObserver(self)
     }
-
+    
     fileprivate func configureOverlayView() {
 
+        /// Make the overlay view occupy the whole screen
```

Comment:
I don't think that's what the next line does. The intention here is to give the overlay enough padding so it still covers the screen when rotated.
Let's get rid of this comment, people will understand what the next line does without it